jeeves/jobs.py

```diff
@@ -126,9 +126,11 @@ def get_jenkins_jobs(server, job_search_fields):
 
 def get_osp_version(job_name):
 	''' gets osp version from job name via regex
-		returns None if no version is found
+		if multiple versions detected, the highest number is considered osp version and
+		returns osp version or None if no version is found
 	'''
-	version = re.search(r'\d+\.*\d*', job_name)
-	if version is None:
+	versions = re.findall(r'\b1{1}[0-7]{1}\.?\d?', job_name)
```

Comment:
This won't work with a job name such as 'somejob-osp13'

Reply:
I will try to fix the regex